touchevents.html

```diff
@@ -555,199 +476,152 @@ <h3 id="webidl-conform">WebIDL Conformance</h3>
+        If a Web application can process touch events, it can <a href="#dfn-canceled-event">cancel</a> the events, and no corresponding mouse events would need to be dispatched by the user agent. If the Web application is not specifically written for touch input devices, it will react to the subsequent mouse events instead.
+      </p>
+
+      <p class="note">
+        User agents will typically dispatch mouse and click events when there is only a single <a>active touch point</a>. Multi-touch interactions – involving two or more <a href="#dfn-active-touch-point">active touch points</a> – will usually only generate touch events.
       </p>
 
       <p id="click-events">
-        If the user agent intreprets a sequence of touch events as a click,
-        then it should dispatch <a>mousemove</a>, <a>mousedown</a>,
-        <a>mouseup</a>, and <a>click</a> events (in that order) at the location
-        of the <a>touchend</a> event for the corresponding touch input.  If the
-        contents of the document have changed during processing of the touch
-        events, then the user agent may dispatch the mouse events to a
-        different target than the touch events.
+        If the user agent interprets a sequence of touch events as a click, then it should dispatch <a><code>mousemove</code></a>, <a><code>mousedown</code></a>, <a><code>mouseup</code></a>, and <a><code>click</code></a> events (in that order) at the location of the <a><code>touchend</code></a> event for the corresponding touch input. If the contents of the document have changed during processing of the touch events, then the user agent may dispatch the mouse events to a different target than the touch events.
       </p>
 
       <p>
-        The default actions and ordering of any further touch and mouse events
-        are implementation-defined, except as specified elsewhere.
+        The default actions and ordering of any further touch and mouse events are implementation-defined, except as specified elsewhere.
       </p>
+
+      <section class="note">
+      <p>The activation of an element (a tap) would typically produce the following event sequence (though this may vary slightly, depending on specific user agent behavior):</p>
+      <ol data-class="note-list">
+          <li><code>touchstart</code></li>
+          <li>Zero or more <code>touchmove</code> events, depending on movement of the finger</li>
+          <li><code>touchend</code></li>
+          <li><code>mousemove</code> <small>(for compatibility with legacy mouse-specific code)</small></li>
+          <li><code>mousedown</code></li>
+          <li><code>mouseup</code></li>
+          <li><code>click</code></li>
+      </ol>
+      <p>If, however, either the <code>touchstart</code>, <code>touchmove</code> or <code>touchend</code> event has been <a href="#dfn-canceled-event">canceled</a> during this interaction, no mouse or click events will be fired, and the resulting sequence of events would simply be:
```

Comment:
Yes, the REC version accounted for bugs in Android browser and Gecko that have since been fixed.  v1-eratta should describe the bug-free version where consuming touchend prevents a click (in practice many sites now rely on this and it is the recommended way of suppressing click without suppressing scroll).